config/tailwind/tailwind.config.js

```diff
@@ -2,6 +2,7 @@ import colors from 'tailwindcss/colors';
 
 /** @type {import('tailwindcss').Config} */
 export const tailwindConfig = {
+  blocklist: ['overflow-scroll'],
```

Comment:
@ayoaru I added `overflow-scroll` to the blocklist to safeguard us from using it in the future!

Reply:
Perfect, do I need to do anything else on my end or am I good to merge?